logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/user/SearchKnownUserUseCaseTest.kt

```diff
@@ -1,242 +1,242 @@
-                            userType = UserType.EXTERNAL
-                        )
-                    )
-                )
-            )
-
-        return this
-    }
-
-    fun withSearchKnownUsersByNameOrHandleOrEmail(
-        searchQuery: String? = null,
-        extraOtherUser: OtherUser? = null
-    ): Arrangement {
-        val otherUsers = listOf(
-            OtherUser(
-                id = QualifiedID(
-                    value = "someSearchQuery",
-                    domain = "wire.com",
-                ),
-                name = null,
-                handle = null,
-                email = null,
-                phone = null,
-                accentId = 0,
-                team = null,
-                connectionStatus = ConnectionState.ACCEPTED,
-                previewPicture = null,
-                completePicture = null,
-                availabilityStatus = UserAvailabilityStatus.NONE,
-                userType = UserType.FEDERATED
-            ),
-        )
-
-        if (extraOtherUser != null) {
-            otherUsers.plus(extraOtherUser)
-        }
-
-        given(searchUserRepository)
-            .suspendFunction(searchUserRepository::searchKnownUsersByNameOrHandleOrEmail)
-            .whenInvokedWith(if (searchQuery == null) any() else eq(searchQuery))
-            .thenReturn(
-                UserSearchResult(
-                    otherUsers
-                )
-            )
-
-        return this
-    }
-
-    fun arrange(): Pair<Arrangement, SearchKnownUsersUseCase> {
-        return this to SearchKnownUsersUseCaseImpl(searchUserRepository, userRepository)
-    }
-}
+//package com.wire.kalium.logic.feature.user
+//
+//import com.wire.kalium.logic.data.id.QualifiedID
+//import com.wire.kalium.logic.data.publicuser.SearchUserRepository
+//import com.wire.kalium.logic.data.publicuser.model.OtherUser
+//import com.wire.kalium.logic.data.user.ConnectionState
+//import com.wire.kalium.logic.data.user.UserAvailabilityStatus
+//import com.wire.kalium.logic.data.user.type.UserType
+//import com.wire.kalium.logic.feature.publicuser.search.SearchKnownUsersUseCase
+//import com.wire.kalium.logic.feature.publicuser.search.SearchKnownUsersUseCaseImpl
+//import io.mockative.Mock
+//import io.mockative.any
+//import io.mockative.anything
+//import io.mockative.classOf
+//import io.mockative.eq
+//import io.mockative.given
+//import io.mockative.mock
+//import io.mockative.verify
+//import kotlinx.coroutines.test.runTest
+//import kotlin.test.Test
+//import kotlin.test.assertIs
+//import com.wire.kalium.logic.data.publicuser.model.UserSearchResult
+//import com.wire.kalium.logic.data.user.UserRepository
+//import com.wire.kalium.logic.feature.publicuser.search.Result
+//import com.wire.kalium.logic.feature.publicuser.search.SearchUsersUseCase
+//import com.wire.kalium.logic.framework.TestUser
+//import kotlin.test.assertFalse
+//
+//class SearchKnownUserUseCaseTest {
```

Comment:
Tests still commented out?

Also, missing tests to assert the new behaviour with `SearchOptions`